### DISCUSSION TITLE
Show tagged items in category blog layout

### DISCUSSION THREAD

**Discussion by WoodyF4u** [joomla/joomla-cms#47039] [2. Cold Features]
### Steps to reproduce the issue
With an adjustment it was possible in Joomla 3 to create a category blog layout for tagged articles. This no longer works in Joomla 4 and 5.
So it cannot be reproduced on this piece.

### Expected result
All this request has been converted into a new functionality in Joomla. It is then possible to create a Tags display via a menu item under Tags, such as in a category blog.
So the articles are not shown in a list as now, but with title, image and intro text.
The result should therefore be that this is offered as a new functionality in the core of Joomla 5.

### Actual result
I'm not a Joomla programmer myself, so I hope someone with that knowledge will build it.

### System information (as much as possible)
Joomla 5

### Additional comments
It was already very well documented for Joomla 3 at the time:
https://joomla.stackexchange.com/questions/13583/how-to-show-tagged-items-in-category-blog-layout
How it should be made is therefore largely already known.
It just needs to be adapted to current Joomla technology.
Because with the explanation for Joomla 3, Joomla 5 produces  errors in the browser.

I hope more people find this request useful and it can become a new part of Joomla.
In my opinion, it is still a missing piece of technology that is currently missing when using tags.

**Comment by brianteeman** [comment]
Did you try using the tag filter present in the category blog menu item?

**Comment by WoodyF4u** [comment]
Hi Brian,
I just came across this topic:
`https://joomla.stackexchange.com/questions/33575/category-blog-view-filter-by-tags
The starter of that topic has exactly the same question. So there are more people who want the same thing I asked here on Github.
The answer in that thread is somewhat hopeful.
In particular, the override on Brian's page is most in line with what I would want.
https://brian.teeman.net/joomla/897-joomla-tags-as-custom-filters

However, it remains a shame that something like this is not standard in such a beautiful CMS like Joomla.
Tags offer so much more flexibility than categories.

So if possible, I would like to see the idea of ​​category blog layout where the visitor can filter the tags themselves and thus view the desired content become part of the core of Joomla.
So without anyone having to create something with template overrides first.

And if it is not added in a next Joomla version, I hereby ask Brian if it is possible to publish the three template override files somewhere so that I can download them.
Then I can at least be sure that I am using properly working files.

**Comment by brianteeman** [comment]
Many people have followed the instructions in my blog post. It is easy to do. Note that it has a serious limitation that it does not support pagination.

**Comment by WoodyF4u** [comment]
OK, I'll try that template override.

But for a structural solution, my question remains to have it added to the core of Joomla 5.

The current addition of selectable tags in the menu item type category blog is not sufficient.
I miss the tags at the top of the page where the visitor can make a desired selection of articles.

**Comment by luktaas** [comment]
Very definitely, a nice solution suggested by Brian.
Can this code also be used in the same site but with different blog categories with different tags per category?
I never used the earlier feature of J3, but I currently have a J5 site with several blog categories and I use the "list-all-tags" menu item.
But this generates a list-view and, in my opinion, cannot filter by category.
So I am also interested in the solution WoodyF4u is asking for.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43631">issues.joomla.org/tracker/joomla-cms/43631</a>.</sub>

**Comment by WoodyF4u** [comment]
Indeed, Brian's template override is very nice.
But as he himself indicates, not yet fully developed. For example, he rightly mentions that pagination is lacking.
I really think a worked solution is a welcome addition to Joomla 5.

To what extent does this feature request have a chance of being included in a next version?

**Comment by brianteeman** [comment]
**I did not propose my code as I knew it did not answer the request**

**Comment by WoodyF4u** [comment]
Good to know the code is correct.
But it does mean that someone always has to make those primal rides first.
I suspect that the layout as a permanent part of Joomla will be desirable for many people.
That is why I am still curious whether this request can be integrated into Joomla.

**Comment by WoodyF4u** [comment]
I still hope that this will become part of the core Joomla.

In the meantime, I have tried a few times, but I am unable to prepare the three files without getting an error message on the site.

Is it still possible that someone somewhere can offer me the three files?

**Comment by chmst** [comment]
> I still hope that this will become part of the core Joomla.

That depends .. if one of the developers wants to do it. 


> In the meantime, I have tried a few times, but I am unable to prepare the three files without getting an error message on the site.
> 
> Is it still possible that someone somewhere can offer me the three files?

Could you please open a new issue for that? And add some information: the complete error message, how we can replicate it - sceenshots are always helpful

**Comment by brianteeman** [comment]
The code on my blog should not be part of core. It serves a limited use case only and us not a generic solution suitable for all.

**Comment by WoodyF4u** [comment]
I will run Brian's code again soon to see if I can create the three files without getting any error messages on the site.

Brian repeatedly indicates that his code is not the most optimal and therefore should not become part of the core.
That's not my question either.

My question is whether a view can be created for Joomla that can show a category blog layout in a menu item where the website visitor can filter on the tags from the chosen category at the top of the page.

Now only a list of articles can be shown.
But it is much nicer for a website visitor if, in addition to the article title, there is also an image and intro text per article.

So that is what I would like to request as a new functionality for Joomla. 
And that this is included in the standard Joomla CMS so that every administrator can easily apply this from the backend without template overrides.
